client/Components/Games/GameLobby.jsx

```diff
@@ -102,6 +102,19 @@ class GameLobby extends React.Component {
             $('#pendingGameModal').modal('show');
             this.setState({ gameState: GameState.PendingGame });
         }
+
+        if(!props.currentGame && this.props.gameId) {
+            const game = props.games.find(x => x.id === this.props.gameId);
```

Comment:
this should just be props.currentGame

Reply:
wouldn't they already be in a game and not need this then?

Reply:
Ah this is one level higher than i thought it was